repo/scaffold/application.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from ramble.appkit import *
+
+
+class Scaffold(ExecutableApplication):
+    """Scaffold benchmark"""
+
+    name = "scaffold"
+
+    tags = ["python"]
+
+    executable(
+        "modules",
+        "ml load rocm/6.2.1 rocmcc/6.2.1-cce-18.0.1a-magic",
+    )
+    executable(
+        "build",
+        "pip install {package_path} --extra-index-url https://download.pytorch.org/whl/rocm6.2",
+        output_capture=OUTPUT_CAPTURE.ALL,
+    )
+    executable(
+        "config",
+        "scaffold generate_fractals --config {package_path}ScaFFold/configs/benchmark_default.yml",
+    )
+    executable(
+        "run",
+        "scaffold benchmark --config {package_path}ScaFFold/configs/benchmark_default.yml",
```

Comment:
I think I should make these two separate workloads. `scaffold generate_fractals` generates the dataset necessary to run `scaffold benchmark`, but it does not need to be ran every single time. Could I have some option for `ramble on` if the user wants to force regenerate the fractals, e.g. run `scaffold generate_fractals`.